parachain/pallets/inbound-queue/src/fixtures.rs

```diff
@@ -3,8 +3,10 @@
 use hex_literal::hex;
 use snowbridge_beacon_primitives::CompactExecutionHeader;
 use snowbridge_core::inbound::{Log, Message, Proof};
+use sp_core::RuntimeDebug;
 use sp_std::vec;
 
+#[derive(Clone, RuntimeDebug)]
 pub struct InboundQueueTest {
```

Comment:
Lets rename this to the describe the message content, ie RegisterAsset, SendToken, etc.

Reply:
https://github.com/Snowfork/snowbridge/pull/1125/commits/278752a161d5aeff6aee531b5f4ecc39631ea636